src/PageNotFound/PageNotfound.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container, Grid } from '@material-ui/core'
+import React from 'react'
+import  PNFImage from '../Media/PageNotFoundImage.jpg'
+
+interface Props {}
+
+function PageNotfound() {
```

Comment:
Let's use Pascal case here for consistency in component names

```suggestion
function PageNotFound() {
```